### DISCUSSION TITLE
codecov.yml ignore not working when counting tracked lines

### DISCUSSION THREAD

**Discussion by MrCroxx** [codecov/feedback#94] [General]
Hi, I'm new to codecov. It's very useful. But I have some problems and cannot figure out what's wrong.

I configured the top section `ignore` in `codecov.yml`. I also checked it in the report it is enabled. like:

<img width="997" alt="image" src="https://github.com/codecov/feedback/assets/22407295/ce44c7c5-940e-45f4-b9a1-cecc6e4f09b1">

But lines under the matched folder are still tracked:

<img width="1150" alt="image" src="https://github.com/codecov/feedback/assets/22407295/54f0936d-ccb5-4770-a457-ab89c699cd5b">

Have I missed something? Thanks for your help!

**Comment by rohan-at-sentry** [comment]
@MrCroxx thanks for using Codecov. I'm closing this discussion because I've opened https://github.com/codecov/feedback/issues/95 where we can help you troubleshoot.
